concrete/css/build/core/app/components.less

```diff
@@ -73,6 +73,25 @@ ul.item-select-list > li > div.item-select-list-content {
   overflow: hidden;
 }
 
+[data-section=customize-results] {
+  ul.item-select-list {
+     li {
+        padding: 8px;
+        margin-bottom: 2px;
+
+      &:hover {
+          background-color: #e7e7e7;
+          border-radius: 4px;
+          transition: .1s linear background-color;
+      }
+    }
+
+    i.ccm-item-select-list-sort {
```

Comment:
@a3020 

The `i` element is not need for CSS selector specificity, but I was matching the selector style written above it:
https://github.com/MrKarlDilkington/concrete5/blob/1aee646d3c4ccd4dc5e4a9c9fc26801fce1072db/concrete/css/build/core/app/components.less#L54